### PR TITLE
✨ Added test for column sum+ tests for validate_df

### PR DESCRIPTION
**Pull request by m-paz**
## Checklist

This PR:
- [x] follows the guidelines laid out in `CONTRIBUTING.md`
- [ ] links relevant issue(s)
- [x] adds/updates tests (if appropriate)
- [ ] adds/updates docstrings (if appropriate)
- [x] adds an entry in `CHANGELOG.md`

### CODE REVIEW

File: tests/unit/test_task_utils.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import prefect
```
Comment (angelika233) [inline tests/unit/test_task_utils.py:7]: could you please remove unused imports

File: viadot/task_utils.py [hunk 2]
```diff
@@ -723,44 +725,70 @@ def validate_df(df: pd.DataFrame, tests: dict = None) -> None:
...
 
-        # dataset_row_count
         if "dataset_row_count" in tests:
             row_count = len(df.iloc[:, 0])
-            max_value = tests["dataset_row_count"]["max"] or 10_000_000
+            max_value = tests["dataset_row_count"]["max"] or 100_000_000
             min_value = tests["dataset_row_count"]["min"] or 0
 
             if (row_count > min_value) and (row_count < max_value):
                 print("[dataset_row_count] passed.")
```
Comment (angelika233) [inline viadot/task_utils.py:749]: remove print or change it to logger

**Review by angelika233 — CHANGES_REQUESTED** [review]
Please update changelog

Only small comments regarding code created before your changes

**Comment by m-paz** [comment]
> Please update changelog
> 
> Only small comments regarding code created before your changes

The changelog already contains the info about this function. It's just tuning.

File: tests/unit/test_task_utils.py (hunk 1 above)
Comment (m-paz) [reply tests/unit/test_task_utils.py]: done

File: viadot/task_utils.py (hunk 2 above)
Comment (m-paz) [reply viadot/task_utils.py]: done

**Review by angelika233 — APPROVED** [review]
looks good

**Pull request merged**